### ISSUE TITLE
Get test suite passing with JRuby and Rails 5

### ISSUE THREAD

**Issue by seanlinsley** [activeadmin/activeadmin#4305]
In #4254 I wasn't able to get JRuby working with Rails 5, so for now I've set it as an allowed failure. The commit history so far:
- 8dbd357f403dd70f327b0a57d7c48e0a9f64bcfe
- 16aba00450ac5689bd3e5548ee39ccd259cbd32c

```
$ bundle exec rake test_with_coveralls
/home/travis/.rvm/rubies/jruby-9.0.4.0/bin/jruby -Irspec-core-00a82883e26f/lib:rspec-support-3c86358e1329/lib rspec-core-00a82883e26f/exe/rspec --pattern spec/unit/\*\*/\*_spec.rb

Coverage may be inaccurate; set "cli.debug=true" ("-Xcli.debug=true") in your .jrubyrc or do JRUBY_OPTS="-d"

DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super. (called from singleton class at activerecord-jdbc-adapter-1.3.19/lib/arjdbc/jdbc/base_ext.rb:13)

Bundler::GemRequireError: There was an error while trying to load the gem 'activerecord-jdbcsqlite3-adapter'.
  block in require at bundler-1.11.2/lib/bundler/runtime.rb:81
              each at org/jruby/RubyArray.java:1560
  block in require at bundler-1.11.2/lib/bundler/runtime.rb:72
              each at org/jruby/RubyArray.java:1560
           require at bundler-1.11.2/lib/bundler/runtime.rb:61
           require at bundler-1.11.2/lib/bundler.rb:99
             <top> at spec/rails/rails-master/config/application.rb:7
           require at org/jruby/RubyKernel.java:939
   block in (root) at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:1
           require at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:302
   load_dependency at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:268
             <top> at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:302
           require at org/jruby/RubyKernel.java:939
   block in (root) at spec/rails/rails-master/config/environment.rb:2
            (root) at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:1
           require at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:302
             <top> at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:268
           require at org/jruby/RubyKernel.java:939
             <top> at rails-8af52c06558d/activesupport/lib/active_support/dependencies.rb:302
              load at org/jruby/RubyKernel.java:957
   block in (root) at spec/rails_helper.rb:106
              each at org/jruby/RubyArray.java:1560
            (root) at spec/unit/abstract_view_factory_spec.rb:1
            (root) at spec/unit/abstract_view_factory_spec.rb:1
            (root) at rspec-core-00a82883e26f/lib/rspec/core/configuration.rb:1
   load_spec_files at rspec-core-00a82883e26f/lib/rspec/core/configuration.rb:1361
   load_spec_files at rspec-core-00a82883e26f/lib/rspec/core/configuration.rb:1359
             <top> at rspec-core-00a82883e26f/lib/rspec/core/runner.rb:106

Coverage report generated for RSpec to coverage. 1097 / 2564 LOC (42.78%) covered.

/home/travis/.rvm/rubies/jruby-9.0.4.0/bin/jruby -Irspec-core-00a82883e26f/lib:rspec-support-3c86358e1329/lib rspec-core-00a82883e26f/exe/rspec --pattern spec/unit/\*\*/\*_spec.rb failed

The command "bundle exec rake test_with_coveralls" exited with 1.

Done. Your build exited with 1.
```

**Comment by seanlinsley** [comment]
https://github.com/jruby/activerecord-jdbc-adapter/issues/700
https://github.com/jruby/activerecord-jdbc-adapter/issues/704

**Comment by timoschilling** [comment]
closed by @deivid-rodriguez in 7addcf6329a119dc718454ebb56d394121d254dc

**Comment by deivid-rodriguez** [comment]
This is not fixed unfortunately. Still waiting for the adapters to catch up. That commit was just to speed up the build by starting the slowest ruby first.
